test/rebay_api_web/controllers/item_controller_test.exs

```diff
@@ -215,6 +215,21 @@ defmodule RebayApiWeb.ItemControllerTest do
 
       assert json_response(conn, 401)["errors"] != %{}
     end
+
+    test "renders error when user is not authorized", %{conn: conn, item: item, user: user} do
+      conn = TestHelpers.valid_session(conn, user)
```

Comment:
How does this `valid_session` affect this test? Both `valid_session` and the manual cookie configuration is kinda throwing me off.

Reply:
This test is for the case where a user is authenticated (so there is a valid session associated with the user), but they're not authorized to delete the item they're requesting to delete. Do you think it would be sufficient to just change the test description to something like "renders error when user is authenticated but not authorized"?

Reply:
Just so I understand correctly: the item belongs to the user, but then bypasses that by providing a fake session_id on delete?

Reply:
Yeah. I'd say invalid session_id rather than fake though, because it would apply to anything that doesn't match the id stored in the session.

Reply:
I see. I was wondering if it would be more clear to create multiple users (A and B), assign the item to User A, and then try to delete with User B. Considering this is the last day of the project, I wouldn't worry too much about it, I just figured it should be called out.

Reply:
I do have a test for that! Line 230. I wrote the first one to test the authorization that makes sure the user making the request is the same user in the session, and then wrote that authorization, then realized I needed an authorization checking if the requester is the item owner, and wrote the other test. But now that I think of it, the latter may have rendered the former irrelevant.